plugin-packs/postcss-preset-env/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changes to PostCSS Preset Env
 
-### Unreleased (patch)
+### Unreleased (minor)
 
-- Updated [`cssdb`](https://github.com/csstools/cssdb) to [`7.7.3`](https://github.com/csstools/cssdb/blob/main/CHANGELOG.md#773-october-05-2023) (patch)
+- Added `@csstools/postcss-gamut-mapping` [Check the plugin README](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-gamut-mapping#readme) for usage details.
```

Comment:
Should there be any mention to the progressive custom properties?

Reply:
A changelog entry for `progressive custom properties` will be generated by the release planner.

I don't think we need to add more detail here because users of `postcss-preset-env` are unlikely to know that this plugin is part of the inner workings :)